src/sentry/tasks/self_hosted/__init__.py

```diff
@@ -0,0 +1,3 @@
+from sentry.taskworker.registry import TaskNamespace, taskregistry
+
+selfhosted_tasks: TaskNamespace = taskregistry.create_namespace("selfhosted")
```

Comment:
this annotation is unnecessary -- mypy has return-value inference

Reply:
also ideally we should not be putting code in `__init__.py` -- it makes it way too easy to introduce import cycles

Reply:
:+1: After some discussion, we're planning on consolidating namespace definitions to a module that isn't `__init__.py`.